packages/react-ui/src/widgets/ChartLegend.js

```diff
@@ -0,0 +1,169 @@
+import React, { useState, useRef, useEffect, useCallback } from 'react';
```

Comment:
Mostly the same component copy / pasted, but adapted to PieWidget

https://github.com/CartoDB/carto-react/pull/767/files#diff-25c0014f515956ba1c9b2abf63cf9aea3ce7458ae32a195710015f038309ce18

Reply:
would be possible to re-use those parts instead of copy&paste?

Reply:
yes, @zbigg already did it in his branch :) we'll only have one file (ChartLegend) at the end of both projects